### DISCUSSION TITLE
What is this user and password?

### DISCUSSION THREAD

**Discussion by williank** [goharbor/harbor#21094] [General]
![image](https://github.com/user-attachments/assets/4b852870-40cf-44bf-90f9-dd20c71790a6)

**Comment by Vad1mo** [comment]
can you be more specific?

**Reply by williank** [reply]
I did the installation and it is not entering with the default login and password.

**Reply by shangmin2009** [reply]
You can check the correct password in harbor.yml by name:
harbor_admin_password

**Reply by williank** [reply]
Where found this file ?
